source/CardGames.py

```diff
@@ -294,45 +291,52 @@ def Play(dealer: Dealer, players: list, pot: Pot):
         players.pop(players.index(player))
       input("Next Player press 'Enter' when ready!")
       print("\n" * 50)
-  return players_passed
+  return [players_passed, pot]
 
-def showWinner(players, pot):
+def showWinner(players, pot, buyIn):
     handTotals = {}
     for player in players:
       handTotals[player.calculateHand()] = player
-    winner = handTotals[max(handTotals.keys())] 
+    winner = handTotals[max(handTotals.keys())]
+    bonus = buyIn - winner.money
+    pot.addPot(bonus) 
     pot.rewardPot(winner)
-    pot.resetPot()
+    print("~~~~~~~~~~~~~~~~~~~~~~~~~~\n $$$ PLAYER TOTALS $$$ \n~~~~~~~~~~~~~~~~~~~~~~~~~~")
+    for player in players:
+      print("%s: $%d" % (player.name, player.money))
+    print("~~~~~~~~~~~~~~~~~~~~~~~~~~")
+    print('\n')
     return winner
 
-source_folder = os.path.join(os.getcwd(), '..', 'source')
-leaderboard_file = os.path.join(source_folder, 'leaderboard.txt')
+leaderboard_file = 'source/leaderboard.txt'
```

Comment:
I'm not sure why you took this portion out. Did you test out the leaderboard functionality with this adjustment?

Reply:
Yes, the leaderboard functionality did not work with that implementation so I had to change it at least on my local machine.